syntax = "proto2";

option java_multiple_files = true;
option java_outer_classname = "ProtoPolygonArrayWrapper";

package org.openstreetmap.atlas.proto;

message ProtoPolygonArray {
    optional string name = 1;
    repeated bytes encodings = 2;
}
